docs/user_guide/faq.rst

```diff
@@ -0,0 +1,112 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    For a high level view of the subject, please see the :ref:`further reading <further_reading>`
+    section, where we provide links to a variety of books and papers.
+    In our :ref:`user guide <user_guide>` we also have links to the papers describing
+    Fairlearn's mitigation algorithms.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
+
+    To elaborate, information is often redundantly encoded in datasets, and machine learning
+    algorithms *will* uncover these links (it is what they are designed to do).
+    For example, in the US, the ZIP code where a person lives is well correlated with their
+    race.
+    Even if the model is not provided with race as a feature, the model will pick up on it
+    implicitly via the ZIP code (and other features).
+    Worse, without having the race available in the dataset, it is not possible to determine
+    whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+The model is unfair because the data are biased. Isn't it better to get better data?
+    If a model is unfair due to biased data (note that this is not the only reason a model
+    may be unfair), then obtaining unbiased data is definitely the best solution.
+    However, doing so might not always be practical.
+    In these cases, the mitigation techniques in Fairlearn and similar packages can redistribute
+    harms between subgroups in the data.
+    The extent to which this is 'fair' will depend on how the harms are redistributed, and the
+    broader context of the system of which the model is a part.
+
+Won't making a model fairer reduce its accuracy?
+    Any technique which reduces disparity in some metric is most likely to impact other metrics.
+    However, we have found empirically that it is usually possible to obtain substantial reductions
+    in disparity (as measured by some suitable metric) at minimal cost in accuracy.
+    Indeed, for certain subgroups in the data, accuracy may be improved dramatically.
+
+Fairlearn questions
+-------------------
+
+Can Fairlearn make my model fair?
+    Naive use of Fairlearn is not going to result in fairer models.
+    Indeed, when used incorrectly, Fairlearn could produce a model which is *less* fair.
+
+    What Fairlearn can do is generate a model which has lower disparity as measured by
+    a user-specified metric.
+    Whether this makes the model fairer is outside the remit of Fairlearn - or indeed of
```

Comment:
I re-read this a few times and I don't like how it portrays the project. I think a big part of what we want to achieve is that Fairlearn is *not* just code, but educational material to help stakeholders understand fairness and apply the toolkit in practice. That would mean it's very much not "ouside the remit of Fairlearn" (unless you mean just the code), but we should rather provide the resources to enable this to be done properly